tests/unit/test_circular_imports.py

```diff
@@ -75,6 +81,8 @@ def test_no_warnings(import_path: str) -> None:
     by circular imports.
 
     DeprecationWarnings related to distutils in ansible_runner are ignored
+
+    :param import_path: The path that should be imported and checked for warnings
```

Comment:
I think this would be a bit more accurate:
```suggestion
    :param import_path: The path to be imported and smoke-checked for warnings and crushes
```